promql/parser/lex.go

```diff
@@ -139,6 +139,7 @@ var ItemTypeStr = map[ItemType]string{
 	COMMA:         ",",
 	EQL:           "=",
 	COLON:         ":",
+	DOUBLECOLON:   "::",
```

Comment:
Although it isn't strictly necessary, I added the `DOUBLECOLON` in order to keep with the theme of having two character groups as one symbol.